source/beamline/IDEAS/IDEASMonochromatorControl.cpp

```diff
@@ -42,8 +42,8 @@ void IDEASMonochromatorControl::onAllControlsConnected()
 AMControl::FailureExplanation IDEASMonochromatorControl::move(double setpoint)
 {
 
-	if (setpoint > IDEASBeamline::ideas()->monoHighEV()->value() + 824)  //HACK for max energy
-		setpoint = IDEASBeamline::ideas()->monoHighEV()->value() + 824;
+	if (setpoint > IDEASBeamline::ideas()->monoHighEV()->value())
```

Comment:
Nope!

Tonia made the HighEV and LowEV PVs writable, so I can change them to reflect reality instead of adding a correction to an arbitrary number.